### PR TITLE
Update YMDK Split 64 config with correct indicator LED pins

### PR DESCRIPTION
**Pull request by minkezhang**
## Description

Added indicator LED pinouts for the YMDK Split64, based on the [PS2AVRGB](https://github.com/showjean/ps2avrU/blob/master/firmware/ps2avrGB/Circuit/ps2avrGB/20140829_140848_001.png) circuit diagram and manually verified.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ymdk/sp64/config.h [hunk 1]
```diff
@@ -30,6 +30,16 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define RIGHT_HALF
 
+/* LED indicator lights
+ *
+ * Based off of the ps2avrGB pearl (/keyboards/pearl) and manually verified.
+ *
+ * See https://github.com/qmk/qmk_firmware/issues/9123#issuecomment-630235214.
+ */
```
Comment (zvecr) [inline keyboards/ymdk/sp64/config.h:38]:
```suggestion
/* LED indicator lights */
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

File: keyboards/ymdk/sp64/rules.mk [hunk 2]
```diff
@@ -10,16 +10,16 @@ BOOTLOADER = bootloadhid
 BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
 MOUSEKEY_ENABLE = no        # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = no        # Console for debug
+CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = yes        # Commands for debug and configuration
 NKRO_ENABLE = no            # Enable N-Key Rollover
-#BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
-AUDIO_ENABLE = no    # There is no speaker on this PCB
+# BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
```
Comment (fauxpark) [inline keyboards/ymdk/sp64/rules.mk:16]:
```suggestion
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
```

File: keyboards/ymdk/sp64/rules.mk [hunk 3]
```diff
@@ -10,16 +10,16 @@ BOOTLOADER = bootloadhid
...
+AUDIO_ENABLE = no           # There is no speaker on this PCB
```
Comment (fauxpark) [inline keyboards/ymdk/sp64/rules.mk:17]:
```suggestion
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ymdk/sp64/rules.mk (hunk 3 above)
Comment (minkezhang) [reply keyboards/ymdk/sp64/rules.mk:17]: I do not see audio output for the PCB in question -- shouldn't there be an explicit warning in the config that this option will do nothing?

Comment (drashna) [reply keyboards/ymdk/sp64/rules.mk:17]: Not really, no need.  Setting it to "no" indicates that.  You can add `AUDIO_SUPPORTED = no` to ensure that.

Mostly, it's part of the default block of rules, that we want to keep consistent.

Comment (minkezhang) [reply keyboards/ymdk/sp64/rules.mk]: Done.

**Review by drashna — APPROVED** [review]

**Pull request merged**